server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/data/CassandraDomainListModule.java

```diff
@@ -59,12 +57,8 @@ public CassandraDomainListConfigurationPerformer(ConfigurationProvider configura
         }
 
         @Override
-        public void initModule() {
-            try {
-                cassandraDomainList.configure(configurationProvider.getConfiguration("domainlist"));
-            } catch (ConfigurationException e) {
-                Throwables.propagate(e);
```

Comment:
here the simpler change is to replace this line by "throw Throwables.propagate(e)". There is just a missing throw